Gems/ROS2/Code/Include/ROS2/Manipulation/MotorizedJoints/JointMotorControllerComponent.h

```diff
@@ -58,5 +59,6 @@ namespace ROS2
 
         // AZ::TickBus overrides
         void OnTick(float deltaTime, AZ::ScriptTimePoint time) override;
+        builtin_interfaces::msg::Time m_lastTickTime;
```

Comment:
Add an empty line to separate it from the `// AZ::TickBus overrides`, add a comment

```suggestion

        builtin_interfaces::msg::Time m_lastTickTime; //!< ROS 2 Timestamp during last OnTick call
```